.orchestra/support/aws-sdk-download-submodules.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+function curl() {
+    if [[ $# != 3 || $2 != "--output" ]]; then
+        return 1
+    fi
+
+    FILENAME="$(basename "$1")"
+    "${ORCHESTRA_DOTDIR}/support/fetch.sh" "$1"
+    mv "$FILENAME" "$3"
+}
+
+export UNZIP="-qq"
+source ./prefetch_crt_dependency.sh
```

Comment:
This file cannot ever be re-used with anything else other than `awk-sdk.yml`.
Let's merge it into `aws-sdk.yml` or let's make a `bash` script called `curl` and let's put in `PATH`: that enables us to do the same with `function curl` + `source`.

Reply:
That's nice, **in theory**, but comes with the added caveat that, in the long run, it'd require us to implement (possibly) a lot of curl command line options. I'd go (for now) with integrating it inside the `aws-sdk` component

Reply:
Merged into `aws-sdk` configure script